CODEOWNERS

```
@@ -436,7 +436,7 @@ homeassistant/components/sisyphus/* @jkeljo
 homeassistant/components/sky_hub/* @rogerselwyn
 homeassistant/components/slack/* @bachya
 homeassistant/components/slide/* @ualex73
-homeassistant/components/sma/* @kellerza @rklomp
+homeassistant/components/sma/* @rklomp @kellerza
```

Suggested change:
```suggestion
homeassistant/components/sma/* @kellerza @rklomp
```
This change is not relevant for the version bump, and there are no priority in CODEOWNERS, when a new person becomes CODEOWNERS, his/hers name is appended.

---

There is a priority in CODEOWNERS for github, as new issues get assigned to the first user in the list. As I am now mainly maintaining the `pysma` library and `sma` integration I want to have any issues assigned to myself.

---

But it has nothing to do with the version bump. Apart from that it is not totally true, see 

homeassistant/components/modbus/* @adamchengtkc @janiversen @vzahradnik
and even though I am second issues gets assigned to me (also before I became member).

---

Interesting. Based on what condition are you assigned automatically to an issue than?
At the moment all `sma` related issues are assigned to @kellerza 
https://github.com/home-assistant/core/issues?q=is%3Aissue+label%3A%22integration%3A+sma%22